### DISCUSSION TITLE
Using next-video player without providers

### DISCUSSION THREAD

**Discussion by nosovandrew** [muxinc/next-video#226] [Q&A] [answered]
My videos are stored in Cloudinary. Could I use just simple next-video player without any providers (like mux, vercel-blob, etc) like below:
```js
import Video from 'next-video';

export default function Page() {
    return (
      <>
          <Video src="https://res.cloudinary.com/<pathtomp4>" />
      </>
    );
}
```

My `next.config.mjs`:
```js
import { withNextVideo } from "next-video/process";
/** @type {import('next').NextConfig} */
const nextConfig = {
    compiler: {
        styledComponents: true,
    },
};

export default withNextVideo(nextConfig);
```

This method works, but I'm not sure about the rightness of this approach. I didn't find any info about this case in docs.

In addition, during compile my app generates empty file `_next-video` in `/public` folder,  I don't know what is it.

**Comment by luwes** [comment, marked as answer]
That's a good question!

The default player supports mp4 files so it should be okay. One thing that might be less ideal is that in development mode only you might see a failed request to `/api/video` which tries to upload and process remote video files (see https://github.com/muxinc/next-video#remote-videos)

The `_next-video` is a temporary symlink and it's gitignored, it can be ignored.
It's there to symlink `/public/_next-video` to the `/videos` folder so local videos can be temporarily played in dev mode.
